examples/commoncrawl/build.gradle

```diff
@@ -0,0 +1,38 @@
+buildscript {
+    repositories {
+        maven {url 'http://distro.marklogic.com/nexus/repository/maven-snapshots/'}
+	mavenLocal()
+        maven { url "https://plugins.gradle.org/m2/" }
+        jcenter()
+    }
+    dependencies {
+        if (project.hasProperty("testing")) {
+            classpath "com.marklogic:ml-data-hub:5.4-SNAPSHOT"
+
+        } else {
+            classpath "com.marklogic:ml-data-hub:5.3.0"
+        }
+    }
+}
+plugins {
+    id "net.saliman.properties" version "1.5.1"
+    id "java"
+}
+apply plugin: "com.marklogic.ml-data-hub"
+
+repositories {
```

Comment:
Since we're not compiling any Java code, you can remove the "java" plugin on line 19 and these repositories and dependencies blocks.